From 849decec3882164df5895a2ddd351c56c15fec2c Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 22 Nov 2016 07:34:45 -0500
Subject: [PATCH 11/21] always use the WebView sandbox

---
 .../webview/chromium/WebViewChromiumFactoryProvider.java | 16 ++--------------
 1 file changed, 2 insertions(+), 14 deletions(-)

diff --git a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
index 2c5777f845b7..da8640843e75 100644
--- a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
+++ b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
@@ -228,20 +228,8 @@ public class WebViewChromiumFactoryProvider implements WebViewFactoryProvider {
 
         CommandLineUtil.initCommandLine();
 
-        boolean multiProcess = false;
-        if (BuildInfo.isAtLeastO()) {
-            // Ask the system if multiprocess should be enabled on O+.
-            multiProcess = mWebViewDelegate.isMultiProcessEnabled();
-        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-            // Check the multiprocess developer setting directly on N.
-            multiProcess = Settings.Global.getInt(
-                    ContextUtils.getApplicationContext().getContentResolver(),
-                    Settings.Global.WEBVIEW_MULTIPROCESS, 0) == 1;
-        }
-        if (multiProcess) {
-            CommandLine cl = CommandLine.getInstance();
-            cl.appendSwitch("webview-sandboxed-renderer");
-        }
+        CommandLine cl = CommandLine.getInstance();
+        cl.appendSwitch("webview-sandboxed-renderer");
 
         ThreadUtils.setWillOverrideUiThread();
         // Load chromium library.
-- 
2.15.0

