From fe5477df08868239a14a38cc76e7bb30fd6a73d6 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 22 Nov 2016 07:34:45 -0500
Subject: [PATCH 12/23] always use the WebView sandbox

---
 .../chromium/WebViewChromiumFactoryProvider.java | 16 ++--------------
 1 file changed, 2 insertions(+), 14 deletions(-)

diff --git a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
index 6514317b5855..f05346fc379b 100644
--- a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
+++ b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
@@ -192,20 +192,8 @@ public class WebViewChromiumFactoryProvider implements WebViewFactoryProvider {
 
         CommandLineUtil.initCommandLine();
 
-        boolean multiProcess = false;
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            // Ask the system if multiprocess should be enabled on O+.
-            multiProcess = mWebViewDelegate.isMultiProcessEnabled();
-        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-            // Check the multiprocess developer setting directly on N.
-            multiProcess = Settings.Global.getInt(
-                    ContextUtils.getApplicationContext().getContentResolver(),
-                    Settings.Global.WEBVIEW_MULTIPROCESS, 0) == 1;
-        }
-        if (multiProcess) {
-            CommandLine cl = CommandLine.getInstance();
-            cl.appendSwitch("webview-sandboxed-renderer");
-        }
+        CommandLine cl = CommandLine.getInstance();
+        cl.appendSwitch("webview-sandboxed-renderer");
 
         ThreadUtils.setWillOverrideUiThread();
         BuildInfo.setBrowserPackageInfo(packageInfo);
-- 
2.17.1

