From efd54a4591488ae0f5678fa68412b812d256f1f4 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 18 Nov 2016 07:29:13 -0500
Subject: [PATCH 16/21] switch to -fstack-protector-strong

---
 build/config/compiler/BUILD.gn | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index fff8488778d7..18eb75320d09 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -207,16 +207,12 @@ config("compiler") {
         cflags += [ "-fstack-protector" ]
       }
     } else if (is_posix && !is_chromeos && !is_nacl) {
-      # TODO(phajdan.jr): Use -fstack-protector-strong when our gcc supports it.
-      # See also https://crbug.com/533294
-      cflags += [ "--param=ssp-buffer-size=4" ]
-
       # The x86 toolchain currently has problems with stack-protector.
       if (is_android && current_cpu == "x86") {
         cflags += [ "-fno-stack-protector" ]
       } else if (current_os != "aix") {
         # Not available on aix.
-        cflags += [ "-fstack-protector" ]
+        cflags += [ "-fstack-protector-strong" ]
       }
     }
 
-- 
2.16.1

