From 548bf3054e34a2558f6a126e61925cf42945f96d Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 22 Dec 2016 07:15:34 -0500
Subject: [PATCH 17/21] enable -fwrapv in non-UBSan builds for clang

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 06c37444bd2f..d2f93123fbd6 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -212,6 +212,10 @@ config("compiler") {
       }
     }
 
+    if (is_clang && !is_ubsan && !is_ubsan_security) {
+        cflags += [ "-fwrapv" ]
+    }
+
     # Linker warnings.
     if (fatal_linker_warnings && !(is_chromeos && current_cpu == "arm") &&
         !(is_android && use_order_profiling) && !is_mac && !is_ios &&
-- 
2.14.1

