From 3ac43ab817c0b27c2d45d273dc4671dc9d5b2117 Mon Sep 17 00:00:00 2001
From: Spotlight <spotlight@joscomputing.space>
Date: Sat, 28 Jul 2018 18:21:36 -0500
Subject: [PATCH 17/22] switch to -fstack-protector-strong

---
 build/config/compiler/BUILD.gn | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index d1bb5597f206..38f49292b2d0 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -257,16 +257,12 @@ config("compiler") {
         cflags += [ "-fstack-protector" ]
       }
     } else if ((is_posix && !is_chromeos && !is_nacl) || is_fuchsia) {
-      # TODO(phajdan.jr): Use -fstack-protector-strong when our gcc supports it.
-      # See also https://crbug.com/533294
-      cflags += [ "--param=ssp-buffer-size=4" ]
-
       # The x86 toolchain currently has problems with stack-protector.
       if (is_android && current_cpu == "x86") {
         cflags += [ "-fno-stack-protector" ]
       } else if (current_os != "aix") {
         # Not available on aix.
-        cflags += [ "-fstack-protector" ]
+        cflags += [ "-fstack-protector-strong" ]
       }
     }
 
-- 
2.19.1

