From 5b153c8ad2da7b95090a49a211c9c66ebb4490c0 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 22 Dec 2016 07:15:34 -0500
Subject: [PATCH 18/23] enable -fwrapv in non-UBSan builds for clang

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 8300e4766d9d..69085d809865 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -262,6 +262,10 @@ config("compiler") {
         cflags += [ "-fstack-protector-strong" ]
       }
     }
+
+    if (is_clang && !is_ubsan && !is_ubsan_security) {
+      cflags += [ "-fwrapv" ]
+    }
   } else {
     cflags += [
       # Assume UTF-8 by default to avoid code page dependencies.
-- 
2.17.1

