From 45655451d10a5443f9c2da692c3dc618428bf56a Mon Sep 17 00:00:00 2001
From: Spotlight <spotlight@joscomputing.space>
Date: Sat, 28 Jul 2018 18:46:25 -0500
Subject: [PATCH 18/22] enable -fwrapv in non-UBSan builds for clang

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 38f49292b2d0..c3766a165991 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -264,6 +264,10 @@ config("compiler") {
         # Not available on aix.
         cflags += [ "-fstack-protector-strong" ]
       }
+
+      if (is_clang && !is_ubsan && !is_ubsan_security) {
+        cflags += [ "-fwrapv" ]
+      }
     }
 
     # Linker warnings.
-- 
2.19.1

