From a98d0777c39f22a18bfe537cbd643bf8c637da63 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 7 Sep 2017 01:50:03 -0400
Subject: [PATCH 24/24] bump Monochrome targetSdkVersion to 26

---
 chrome/android/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 142a806234a8..d1b16cae81d1 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -72,7 +72,7 @@ jinja_template("monochrome_public_android_manifest") {
   output = monochrome_public_android_manifest
   variables =
       chrome_public_jinja_variables +
-      monochrome_android_manifest_jinja_variables + [ "target_sdk_version=24" ]
+      monochrome_android_manifest_jinja_variables + [ "target_sdk_version=26" ]
 }
 
 jinja_template("chrome_sync_shell_android_manifest") {
-- 
2.14.1

